### PR TITLE
Report website issue view (bug 1161864)

### PR DESCRIPTION
**Pull request by mstriemer**
This is mobile only for now (no modal) since we don't support desktop yet.

report-issue mov

### CODE REVIEW

File: src/media/js/views/website/issue.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+define('views/website/issue',
```
Comment (ngokevin) [inline src/media/js/views/website/issue.js]: how is this being required?

**Comment by ngokevin** [comment]
r+wc

File: src/media/js/views/website/issue.js (hunk 1 above)
Comment (mstriemer) [inline src/media/js/views/website/issue.js]: By the router.

**Pull request merged**